src/explorer/model/FavoriteCommand.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { TreeItem } from "vscode";
+import { ITreeItem } from "./ITreeItem";
+import { MavenProject } from "./MavenProject";
+
+export class FavoriteCommand implements ITreeItem {
+    
+    constructor(public project: MavenProject, public command: string, public alias: string, public debug?: boolean) {}
+
+    getContextValue(): string {
+        return "maven:favorites";
+    }
+
+    getTreeItem(): TreeItem | Thenable<TreeItem> {
+        const treeItem: vscode.TreeItem = new vscode.TreeItem(this.alias, vscode.TreeItemCollapsibleState.None);
```

Comment:
would be better if it displays real 'command' along with 'alias', to distinguish from each other, as below.
```
treeItem.description = this.command;
```

<img width="340" alt="image" src="https://user-images.githubusercontent.com/2351748/200212363-3e91beeb-99cc-484f-a50d-160af54682b0.png">